src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -49,7 +49,7 @@ trait RedisTrait
         'failover' => 'none',
         'ssl' => null, // see https://php.net/context.ssl
     ];
-    private \Redis|Relay|\RedisArray|\RedisCluster|\Predis\ClientInterface $redis;
+    protected \Redis|Relay|\RedisArray|\RedisCluster|\Predis\ClientInterface $redis;
```

Comment:
This is a no-go. It would make the property protected in the RedisTagAwareAdapter.

The RedisTagAwareAdapter can already access the private property of the trait as it is the class using the trait.

Reply:
agreed, carless mistake from my side